### DISCUSSION TITLE
wie 'exportDIR' eingeben?

### DISCUSSION THREAD

**Discussion by dorianiro** [mipastgt/AIPBrowserDE-Public#17] [Q&A] [answered]
ich hab es so probiert:
`"exportDIR": "D:\\Flug\\Karten\\AIPBrowserDE\\ExportDir",
`
aber der output landet immer im Standardpfad.  Anlegen des Ordners hatte auch nix gebracht.
Danke und guten Flug!

**Comment by mipastgt** [comment]
Ich habe es sicherheitshalber selbst mal eben probiert. Die folgende Konfigurationsdatei hat bei mir problemlos funktioniert:
```
{
    "version": "1.0.2",
    "name": "Flying in the Ländle",
    "description": "An area covering Baden-Württemberg",
    "noZIP": true,
    "exportDir": "/Users/mpaus/Downloads/tmp",
    "charts": {
        "coveredArea": [
            {
                "type": "GeoRectangle",
                "west": 7.49961623,
                "south": 47.53058632,
                "east": 10.49522706,
                "north": 49.79159732
            }
        ],
        "minInvScale":       1.0,
        "maxInvScale": 1000000.0,
        "imageFormat": "vfrnav",
        "preTransformImages": true
    }
}
```
Wichtig ist dabei bei `"exportDir"` auf die Schreibweise zu achten und den `:` und das `,` am Ende der Zeile nicht zu vergessen.

**Comment by dorianiro** [comment]
In der Tat hatte ich exportDir falsch geschrieben, und das mit dem nicht-backslash das muß man wissen....   leider geht es mit einem Laufwerksbuchstaben trotzdem nicht:
"exportDir": "D:/Flug/Karten/AIPBrowserDE/ExportDir",

Wie muß man das mit dem LW-Buchstaben machen?
Anscheinend muß der Ordner auch vorher angelegt worden sein, aber das macht ja nichts.
Danke!
Volkmar

**Comment by mipastgt** [comment]
Sofern der Pfad existiert, sollten folgende Schreibweisen gleichwertig sein:
```
    Windows: "exportDir" : "C:\Users\ich\test"
    Mac,Linux: "exportDir" : "/home/ich/test"
```
Java ist da auch tolerant was den Slash angeht. Ich habe gerade kein Windows laufen, sonst würde ich das auch noch schnell probieren.

**Comment by dorianiro** [comment, marked as answer]
jetzt hab ichs.
Dein Vorschlag führte unter Windows zu:
Could not export trip-kit  [... ]
`Unexpected JSON token at offset 162: Invalid escaped char 'F' at path: $.version JSON input: .....oZIP": "TRUE", 	"exportDir": "D:\Flug\Karten\AIPBrowserDE\Ex.....`

... was ja darauf schließen läßt daß der 1. backslash als Versuch erkannt wurde, das darauffolgende 'F' zu escapen.
Dann hab ichs erneut mit 2 backslashs probiert, jetzt gehts, so:
	"exportDir": "D:\\Flug\\Karten\\AIPBrowserDE\\ExportDir",
..also kam der Fehler garnicht vom Pfad, sondern weil ich zunächst trotz Wissen um case-sensitiv exportDIR  statt exportDir  geschrieben hatte.
Danke Dir!
Vielleicht die Anleitung ergänzen was die Schreibweise der Pfadangaben angeht, mit Hinweis: Groß/Kleinschreibung beachten?  Jemand ohne IT-Affinität  würde auf die Idee mit den 2 '\\'  wahrscheinlich nicht kommen.  
Danke nochmal!    Volkmar
... ich sehe fast keine posts in Github.   Wird Dein Programm nur selten verwendet?  Wäre schade!

**Comment by mipastgt** [comment]
Gut das dieses Problem jetzt geklärt ist :-)
Zur Benutzung der Software: Mehr Werbung könnte natürlich nicht schaden (!!!) aber die meisten Rückmeldungen erhalte ich tatsächlich immer noch per E-Mail. Die Hürde sich erst bei GitHub registrieren zu müssen scheint für viele zu hoch zu sein. In anderen Foren etc. kann man i.d.R. aber auch nicht posten ohne sich zu registrieren.

**Reply by dorianiro** [reply]
Danke nochmal!
